multivac/gather_data.py

```diff
@@ -317,13 +341,20 @@ def put_to_db_test(self):
         for job_id in filter(
                 lambda x: 'failed_tests' in list(self.gathered_data[x].keys()),
                 list(self.gathered_data.keys())):
-            for test in self.gathered_data[job_id].get('failed_tests'):
+            job_info = self.gathered_data[job_id]
+            for test in job_info.get('failed_tests'):
                 tags = {
-                    'job_id': job_id,
                     'configuration': test['conf'],
-                    'job_name': self.gathered_data[job_id]['job_name'],
-                    'commit_sha': self.gathered_data[job_id]['commit_sha'],
-                    'test_attempt': test['test_attempt']
+                    'test_type': test['test_type'],
+                    'diff_test_subtype': test['subtype'],
+                    'debug': job_info['debug'],
+                    'job_id': job_id,
+                    'job_name': job_info['job_name'],
+                    'commit_sha': job_info['commit_sha'],
+                    'test_attempt': test['test_attempt'],
+                    'architecture': job_info['platform'],
+                    'gc64': job_info['gc64'],
+                    'os_version': job_info['runner_label'][0],
```

Comment:
are you sure that the first label is always the OS?

Reply:
There are always only one label, as I can see